eodhp-web-presence/templates/web-presence-deploy.yaml

```diff
@@ -12,66 +12,32 @@ spec:
       labels:
         app: eodhp-web-presence
     spec:
-      serviceAccountName: {{ .Values.serviceAccount.name | default "default" }}
+      serviceAccountName: {{ .Values.web.serviceAccount.name }}
       containers:
         - name: eodhp-web-presence
-          image: {{ .Values.image | default "public.ecr.aws/n1b3o1k2/eodhp-web-presence:0.1.12" }}
+          image: {{ .Values.web.image }}
           imagePullPolicy: Always
           ports:
             - containerPort: 8000
+          envFrom:
+            {{- toYaml .Values.web.extraEnvFrom | nindent 12 }}
           env:
-            - name: APP_URL
-              value: {{ .Values.app_url | default "www.example.com" }}
-            - name: SQL_HOST
-              value: {{ .Values.database.sql_host | default "postgres-service" }}
-            - name: SQL_PORT
-              value: {{ .Values.database.sql_port | default "5432" | quote }}
-            - name: SQL_DATABASE
-              value: {{ .Values.database.sql_database | default "web_presence_db" }}
-            - name: SQL_ENGINE
-              value: {{ .Values.database.sql_engine | default "django.db.backends.postgresql_psycopg2" }}
-            - name: SQL_USER
-              value: {{ .Values.database.sql_user | default "admin" }}
-            {{- if and (.Values.database.secretKeys.sqlPasswordKey) (.Values.database.existingSecret) }}
+            {{- if and (.Values.web.database.secretKeys.sqlPasswordKey) (.Values.web.database.existingSecret) }}
             - name: SQL_PASSWORD
               valueFrom:
                 secretKeyRef:
-                  name: {{ .Values.database.existingSecret }}
-                  key:  {{ .Values.database.secretKeys.sqlPasswordKey }}
+                  name: {{ .Values.web.database.existingSecret }}
+                  key:  {{ .Values.web.database.secretKeys.sqlPasswordKey }}
             {{- else }}
```

Comment:
See previous comment about not requiring the else statement.

Reply:
Removed